content/php-client/resources/target-market-settings/measurement-families.md

```diff
@@ -0,0 +1,116 @@
+### Measurement family
+
+#### Get a list of measurement families
+::: php-client-availability versions=6.0 editions=CE,EE
+
+```php
+$client = new \Akeneo\Pim\ApiClient\AkeneoPimClientBuilder('http://akeneo.com/')->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+
+/*
+ * Returns an array like this:
+ * [
+ *     [
+ *         'code' => 'Angle',
+ *         'labels' => [
+ *             'en_US' => 'Angle',
+ *             'fr_FR' => 'Angle',
+ *         ],
+ *         'standard_unit_code' => 'RADIAN',
+ *         'units' => [
+ *             'RADIAN' => [
+ *                 'code' => 'RADIAN',
+ *                 'labels' => [
+ *                     'en_US' => 'Radian',
+ *                     'fr_FR' => 'Radian',
+ *                 ],
+ *                 'convert_from_standard' => [
+ *                     [
+ *                         'operator' => 'mul',
+ *                         'value' => '1',
+ *                     ],
+ *                 ],
+ *                 'symbol' => 'rad',
+ *             ],
+ *             'MILLIRADIAN' => [
+ *                 'code' => 'MILLIRADIAN',
+ *                 'labels' => [
+ *                     'en_US' => 'Milliradian',
+ *                     'fr_FR' => 'Milliradian',
+ *                 ],
+ *                 'convert_from_standard' => [
+ *                     [
+ *                         'operator' => 'mul',
+ *                         'value' => '0.001',
+ *                     ],
+ *                 ],
+ *                 'symbol' => 'mrad',
+ *             ],
+ *         ],
+ *     ],
+ *     ...
+ * ]
+ */
+$measurementFamilies = $client->getMeasurementFamilyApi()->all();
+```
+
+You can get a complete description of the returned format [here](/api-reference.html#measurement_families_get_list).
+
+::: warning
+There is no pagination on measurement families.
+:::
+
+#### Upsert a list of measurement families 
+::: php-client-availability versions=6.0 editions=CE,EE
+
+This method allows to create or update a list of measurement families.
```

Comment:
```suggestion
This method allows to create or partial update a list of measurement families.
```
?

Reply:
Same here, this is the exact wording used everywhere else.
The partial update behavior on PATCH is the same for all the endpoints, as documented here: https://api.akeneo.com/documentation/update.html